### DISCUSSION TITLE
Strike through item in incremental bullleted list at end of list

### DISCUSSION THREAD

**Discussion by jpmorr** [quarto-dev/quarto-cli#10670] [Q&A]
### Description

I'm trying to strike out an item in an incremental bulleted list in a revealjs presentation at the end of the slide (after going through all of the bulleted items) but I can't quite figure out the syntax using fragments. 

This seems like it should be really simple but the closest I've managed are the two examples below which are not quite right. One missing a bullet and the other in the wrong order.

````qmd
---
title: "Quarto Strikethrough Test"
format: revealjs
---

## strikethrough test 1

::: {.fragment .nonincremental fragment-index=1}
 - point one 
:::

::: {.fragment fragment-index=2}
::: {.fragment fragment-index=4 .strike}
 point two 
:::
:::

::: {.fragment .nonincremental fragment-index=3}
 - point three 
:::

## strikethrough test 2

 - point one 
 - [point two ]{.fragment .strike}
 - point three 

````

This shouldn't be so complicated - what am I getting wrong in the nesting or fragment ordering?

**Comment by mcanouil** [comment]
What is supposed to be `.strike` class? That does not exist.
And what is supposed to happen exactly? Should the text be strike out after fragment?

**Comment by biofe** [comment]
Hi,

I struggled with this as well and - thanks ChatGPT - found a solution with HTML that works as desired:

**Create `reveal-extra.html` with this content:**

```HTML
<script>
Reveal.on('fragmentshown', event => {
  if (event.fragment.classList.contains('strike-trigger')) {
    document.querySelectorAll('.strike-target')
      .forEach(el => el.classList.add('struck'));
  }
});
</script>
```

**Add it to your YAML options:**

```YAML
format:
  revealjs:
    include-after-body: reveal-extra.html
```

**Use it as follows:**

```Markdown
## Strike through test

<ul>
  <li class="fragment" data-fragment-index="1">
    First point
  </li>

  <li class="fragment strike-target" data-fragment-index="2">
    Second point
  </li>

  <li class="fragment" data-fragment-index="3">
    Third point
  </li>
</ul>

<span class="fragment strike-trigger" data-fragment-index="4"></span>
```
